File: leetcode-submissions/0215-kth-largest-element-in-an-array/solution.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        Queue<Integer> heap = new PriorityQueue<>();
        for (int num: nums) {
            heap.add(num);
            if (heap.size() > k) {
                heap.poll();
            }
        }
        return heap.poll();
    }
}

File: leetcode-submissions/0215-kth-largest-element-in-an-array/solution.py
import heapq
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        
        i = 0
        n = len(nums)
        heap = []
        
        for i in range(n):      
            heapq.heappush(heap, nums[i])
            
            if len(heap) > k :
                heapq.heappop(heap)
        
        return heap[0]


